miden-crypto/src/merkle/smt/partial.rs

```diff
@@ -248,7 +248,7 @@ impl PartialSmt {
 
     /// Returns an iterator over the [`InnerNode`] and the respective [`NodeIndex`] of the
     /// [`PartialSmt`].
-    pub fn inner_node_indicies(&self) -> impl Iterator<Item = (NodeIndex, InnerNode)> + '_ {
+    pub fn inner_node_indices(&self) -> impl Iterator<Item = (NodeIndex, InnerNode)> + '_ {
```

Comment:
so this was added in https://github.com/0xMiden/crypto/pull/494 to support https://github.com/0xMiden/miden-node/pull/1158, but (for now?) the latter doesn't use these getters.

Reply:
Yes, we still need to propagate the dependencies to miden base/node.